frameworks/cassandra/tests/test_recovery.py

```diff
@@ -28,7 +28,7 @@ def configure_package(configure_security):
 
 
 @pytest.mark.sanity
-@sdk_utils.dcos_1_9_or_higher  # dcos task exec not supported < 1.9
+@pytest.mark.dcos_min_version('1.9')  # dcos task exec not supported < 1.9
```

Comment:
Would it make sense to add a message as a second argument to `mark.dcos_min_version` (is this even possible?) so that we can provide a reason instead of having to add this in a comment? This can then be printed in the `skip` output.

Reply:
I'll add it. Good idea. Much better than comments.